docs/source/models/risk_exposures/child_growth_failure/wasting_state_exposure/index.rst

```diff
@@ -173,6 +172,15 @@ For wave I of the :ref:`nutrition optimization model <2021_concept_model_vivariu
 models for these affected causes will draw from the corresponding GBD 2019 models
 until we update the entire simulation to GBD 2021 results.
 
+.. important::
+
+  We will model wasting transitions as detailed on this page **only** among simulants at least six months of age.
+
+  There will be separate wasting exposure models for simulants 0-6 months of age will be detailed separately.
```

Comment:
or is it simulants 1-6 months?

Reply:
GBD does have CGF exposure for the first month of life, so we could do 0-6... I am not sure yet! Either way, we'll likely have to figure out some sort of initialization plan (maybe birth prevalence?) that I think will be handled on that separate page, so I think saying 0-6 months here is fine at this point.